src/components/spinLoader/spinLoader.module.scss

```diff
@@ -14,23 +14,15 @@
   border-radius: 50%;
   flex-shrink: 0;
   position: relative;
+  mask: radial-gradient(circle, transparent 55%, var(--rp-ui-color-primary) 0%);
 
   &.color-topaz {
-    background: conic-gradient(#0000 0%, var(--rp-ui-base-topaz));
+    background: conic-gradient(#0000 0%, var(--rp-ui-color-primary));
```

Comment:
I just realized that we have a `#0000` color here, can it be replaced with a variable as well?